DESCRIPTION

```diff
@@ -41,8 +41,8 @@ Roxygen: list(markdown = TRUE)
 VignetteBuilder: knitr
 Remotes:
     mrc-ide/conan,
-    mrc-ide/context,
-    mrc-ide/queuer,
+    mrc-ide/context@mrc-4177,
+    mrc-ide/queuer@mrc-4177,
```

Comment:
```suggestion
    mrc-ide/context,
    mrc-ide/queuer,
```

once https://github.com/mrc-ide/context/pull/21 and https://github.com/mrc-ide/queuer/pull/31 are merged (both approved but unmerged at present)